Formula/jdep.rb

```diff
@@ -0,0 +1,104 @@
+class Jdep < Formula
+  desc "Analyze Java .class files and produce Makefile dependencies."
+  homepage "http://www.fudco.com/software/jdep.html"
+  url "http://www.fudco.com/software/jdep-1.3.tar.gz"
+  sha256 "3dd925ae2523cca59e4aaa7cbb8daa28927b40520f0f8dccead91d9edea1dc65"
+
+  patch :DATA
+
+  def install
+    system "make"
```

Comment:
> @DomT4 @MikeMcQuaid Do you want it to function like automake/configure and install to ${PREFIX} ?

Yes.

> You guys should really update the new formula cookbook with all of these requirements.

We cannot update with every corner case as, as in this case, it's a conversation rather than a hard rule.